rosetta-mainnet/elrond-proxy

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14.9 as builder
```

Comment:
Perhaps also here we can use the `folder/Dockerfile` approach. @AdoAdoAdo? And for the testnet as well.

Reply:
I also suggested to go for folder/Dockerfile for both

Reply:
done.